### ISSUE TITLE
Rake Tasks Disabled

### ISSUE THREAD

**Issue by TravisSperry** [rollbar/rollbar-gem#466]
I'm currently getting the following error:
`Rollbar is disabled for Rake tasks since your Rake version is under 0.9.x. Please upgrade to 0.9.x or higher.`
I'm currently running `rake (11.1.2)` and `ruby 2.3.1`

**Comment by jondeandres** [comment]
Hi @TravisSperry, we'll take a look at this. Thanks

**Comment by jondeandres** [comment]
@TravisSperry can you add `require 'rake'` before `Rollbar.configure`? Confirm us this and we'll write a fix for this.

**Comment by TravisSperry** [comment]
Confirmed. That works.

**Comment by radar** [comment]
I am getting this error at the moment still, running on Rake 11.1.2. I get this issue when I run `bundle exec sidekiq`, and when launching `rails console` and it is happening because `Rake` is not loaded at either point. I believe that the `rake.rb` file in Rollbar should attempt to load Rollbar, and do nothing if it can't find it:

```ruby
begin
  require 'rake'
rescue LoadError
end
```

This way, the constant will be loaded if Rake is present in the load path.

**Comment by jondeandres** [comment]
@radar you are right, we pushed this commit https://github.com/rollbar/rollbar-gem/commit/779c25170080b837682856ff2a80cd7a3ac19935 but we removed those `require_dependency` for some issues. But I think we need to add them again and fix those issues.

Let me work on this and I'll reach you ok?

Thanks!

**Comment by radar** [comment]
Sounds good to me, thanks @jondeandres!

**Comment by grosser** [comment]
@TravisSperry did you just want to get rid of the warning or were you actually trying to use rake with rollbar without using `rake` ?

**Comment by TravisSperry** [comment]
Just the warning.

Travis Sperry
travis.sperry1@gmail.com
